tutorials/nlp/Text_Classification_Sentiment_Analysis.ipynb

```diff
@@ -187,6 +174,7 @@
         "os.makedirs(WORK_DIR, exist_ok=True)\n",
         "os.makedirs(DATA_DIR, exist_ok=True)\n",
         "\n",
+        "! wget \"https://firebasestorage.googleapis.com/v0/b/mtl-sentence-representations.appspot.com/o/data%2FSST-2.zip?alt=media&token=aabc5f6b-e466-44a2-b9b4-cf6337f84ac8\" --output-document SST-2.zip\n",
```

Comment:
If we are downloading data, do we need to download its license too (usually thats the case with most datasets)

Reply:
It should not be there actually. Dopped it.